src/Apps/Order/Routes/__tests__/Status.test.tsx

```diff
@@ -1,32 +1,56 @@
 import { Message } from "@artsy/palette"
-import { mount, render } from "enzyme"
+import { render } from "enzyme"
 import React from "react"
-import { HeadProvider } from "react-head"
-import { Status_order } from "../../../../__generated__/Status_order.graphql"
+import { graphql } from "react-relay"
 import {
+  mockResolver,
   OrderWithShippingDetails,
   PickupOrder,
 } from "../../../../Apps/__test__/Fixtures/Order"
-import { StatusRoute } from "../Status"
+import { MockBoot, renderRelayTree, UntilCallback } from "../../../../DevTools"
+import { StatusFragmentContainer } from "../Status"
+
+jest.unmock("react-relay")
 
 describe("Status", () => {
-  const getWrapper = (order: Status_order, headTags = []) =>
-    mount(
-      <HeadProvider headTags={headTags}>
-        <StatusRoute order={order} />
-      </HeadProvider>
-    )
+  const getWrapper = (
+    order: any,
+    headTags: Array<React.ReactElement<any>> = [],
+    until: UntilCallback = null
+  ) => {
+    return renderRelayTree({
+      until,
+      Component: StatusFragmentContainer,
+      query: graphql`
+        query StatusQuery {
+          order(id: "42") {
+            ...Status_order
+          }
+        }
+      `,
+      mockResolvers: mockResolver(order),
+      wrapper: renderer => (
+        <MockBoot breakpoint="xs" headTags={headTags}>
+          {renderer}
+        </MockBoot>
+      ),
+    })
+  }
 
   it("should should have a title containing status", async () => {
-    const headTags = []
-    getWrapper(OrderWithShippingDetails, headTags)
-    // @ts-ignore
-    expect(render(headTags).text()).toBe("Order status | Artsy")
+    const headTags: Array<React.ReactElement<any>> = []
+    await getWrapper(
+      OrderWithShippingDetails,
+      headTags,
+      () => headTags.length > 0
```

Comment:
@zephraph @damassi I now realise I need to dive into this a bit more. I had previously assumed that the title not showing up immediately was due to how `react-head` works, but am now thinking it’s because of our `Responsive` component. I thought that with an initial breakpoint set the `Responsive` component would immediately render, but maybe not?

Reply:
There's a lot going on inside of react-head considering how it works. What makes you think the issue is with Responsive?

Reply:
Yeah the new context API is built around Fiber, so while I haven’t traced it I’m pretty sure it’s async https://github.com/facebook/react/blob/master/packages/react-reconciler/src/ReactFiberNewContext.js

Hrmm 🤔 I think we should follow up in another PR, if we find a way to deal with that at all.

Reply:
I’m gonna leave it at this for now https://twitter.com/alloy/status/1047199392581767168, we’ll see if we hear back or if we can come up with another solution after sleeping on it.

In any case, this shouldn’t block the PR.

Reply:
> What makes you think the issue is with Responsive?

I recall not seeing log statements showing up when I expected them to. I should probably just create a test that uses a context to verify.